frontend/packages/dev-console/integration-tests/features/helm/actions-on-HelmRelease.feature

```diff
@@ -7,15 +7,35 @@ Background:
     And helm release "nodejs-ex-k" is present in topology page
 
 
+@regression, @smoke
+Scenario: Context menu options of helm release: HR-07-TC01
+    Given helm release "nodejs-ex-k" is present in topology page
+    When user right clicks on the helm release "nodejs-ex-k"
+    Then user is able to see the context menu with actions Upgrade, Rollback and Uninstall Helm Release
+
+
+@regression, @smoke
+Scenario: Actions drop down on the side bar: HR-10-TC08
+    Given user is on the topology sidebar of the helm release "nodejs-ex-k"
+    When user clicks on the Actions drop down menu
+    Then user is able to see the actions dropdown menu with actions Upgrade, Rollback and Uninstall Helm Release
+
+
+@regression, @smoke
+Scenario: Actions menu on Helm page
+    Given user is on the Helm page with helm release "nodejs-ex-k"
+    When user clicks on the Kebab menu
+    Then user is able to see kebab menu with actions Upgrade, Rollback and Uninstall Helm Release
+
+
 @regression, @smoke
 Scenario: Perform Upgrade action on Helm Release through Context Menu: HR-08-TC02
     Given user is at the Topolgy page
     When user right clicks on the Helm Release "nodejs-ex-k" to open the context menu
     And user clicks on the "Upgrade" action
-    And user upgrades the chart Version
+    And user upgrades to the chart Version "0.2.0 / App Version 1.16.0 (Provided by Redhat Helm Repo)"
```

Comment:
This statement doesn't need test data, as versions are keep on changing for helm charts. Sorry for confusion

Reply:
Updated